playbooks/roles/airship-configure-ceph/tasks/main.yml

```diff
@@ -41,24 +49,29 @@
   tags:
     - always
 
-#TODO JG move to ceph provisioner chart
-- name: Create ceph secrets yaml
-  template:
-    src: "suse-define-ses-secrets-in-k8s.yml.j2"
-    dest: "/tmp/ceph-secrets.yaml"
-    mode: "0600"
-
-- name: Create ceph storage class yaml
-  template:
-    src: "suse-define-ses-storageclasses-in-k8s.yml.j2"
-    dest: "/tmp/ceph-storage-classes.yaml"
-    mode: "0600"
+- name: Create secrets and StorageClass for external Ceph
```

Comment:
We need to move this whole block (not just apply_on_k8s task)
into common-deployer parts so we do not have the same templates on 2 different places...

Reply:
agreed!

Reply:
I'll look into it